Cargo.toml

```
@@ -42,12 +42,14 @@ serde_json = "1"
 [build-dependencies]
 heck = "0.4"
 pbjson-build = { version = "0.5", optional = true }
+prettyplease = "0.1.25"
 prost-build = { version = "0.11", default-features = false }
 prost-types = "0.11"
 prost-wkt-build = { version = "0.4", optional = true }
 protobuf-src = { version = "1", optional = true }
 schemars = "0.8"
 serde_yaml = "0.9"
+syn = "1.0.109"
```

We just need the `parsing` feature of `syn`:
```suggestion
syn = { version = "1.0.109", default-features = false, features = ["parsing"] }
```